### PR TITLE
chore: release 2024.05.07

### PR DESCRIPTION
**Pull request by denobot**
The following updates are detected:

| module   | from    | to      | type  |
|----------|---------|---------|-------|
|assert|0.224.0|0.225.1|minor|
|bytes|0.224.0|1.0.0-rc.1|prerelease|
|cli|0.224.0|0.224.1|patch|
|collections|0.224.0|0.224.1|patch|
|log|0.224.0|0.224.1|patch|

Please ensure:
- [x] Versions in deno.json files are updated correctly
- [x] Releases.md is updated correctly

The following commits are ignored:

- [chore: fix permissions for version_bump.yml (#4688)](/denoland/deno_std/commit/14f9cc6dbf4dd003916c7c0c7c43a3d248a10336)
- [chore: update version_bump.yml (#4687)](/denoland/deno_std/commit/8426f14e9da673a51c88284321577c5f284e2f68)
- [chore: update `.editorconfig` (#4685)](/denoland/deno_std/commit/cd3bb21d1467abd5fdc7ca2099865e9f49bb57a5)
- [chore: update `title.yml` (#4684)](/denoland/deno_std/commit/cd4f516666a663ce72acf3ade69ff9180695fd07)
- [chore: remove doc test workaround (#4675)](/denoland/deno_std/commit/2f593415d413bbf149840a84579bddac7ca8433a)
- [chore: run `@example` JSDoc code snippets (#4669)](/denoland/deno_std/commit/4bdefc105cdf482ef2043b21562c6bfcd7b0d6a7)
- [chore: add `@template` JSDoc tag check (#4667)](/denoland/deno_std/commit/b884c7ce7348e41795d16564dbb79d434feb7c5e)
- [chore: skip yanked versions (#4660)](/denoland/deno_std/commit/8c9e96c402a92ddebc7f024cfb050dfa8c5b1916)
- [chore: update `check_deprecation.ts` (#4657)](/denoland/deno_std/commit/77996a9eb15dca66ac42709dd7490cfd205b1dd1)
- [chore: update `check_circular_package_dependencies.ts` for workspaces (#4653)](/denoland/deno_std/commit/1c1403a3f19f4dcc40ace47215caf544e6e249a4)
- [docs: update README for JSR (#4654)](/denoland/deno_std/commit/10b8a70a6cbd10e20cd44e2b2905e54bfa5fae03)

---

To make edits to this PR:

```sh
git fetch upstream release-2024-05-07-09-38-51 && git checkout -b release-2024-05-07-09-38-51 upstream/release-2024-05-07-09-38-51
```

### CODE REVIEW

File: Releases.md [hunk 1]
```diff
@@ -1,3 +1,47 @@
+### 2024.05.07
+
+#### @std/assert 0.225.1 (minor)
+
+- test(assert): improve test coverage (#4679)
+- chore(assert,fmt,fs,internal,path): skip yanked versions manually (#4683)
```
[the remark this answers is not in the document]
Comment (kt3k) [reply Releases.md:6]: Ok. Let's skip

**Comment by kt3k** [comment]
Cancelled the version upgrades of `fmt`, `fs`, `internal`, `path` as they don't include anything new.

**Comment by kt3k** [comment]
Currently I set `chore:` tag for release PR title, but I wonder if it makes more sense to tag it like `release: 2024.05.07`. What do you think?

**Comment by iuioiua** [comment]
I don't know how that affects things. It may be easier to find release PRs. I'm mostly impartial.

**Review by kt3k — APPROVED** [review]

**Comment by kt3k** [comment]
Landing

**Pull request merged**